js/long-form.js

```diff
@@ -44,19 +44,24 @@
         })
     }
 
-    var throttleScroll = function(){
+    function debounce(call_back, wait, this_argument) {
 
-        var scheduled = false;
-        window.addEventListener('scroll', function() {
+        var timer = null;
 
-            if (! scheduled) {
-                scheduled = true;
-                setTimeout(function(){
-                    scheduled = false;
-                    console.log('scrollCount');
-                }, 1000)
-            }
-        });
+        return function (...args) {
+
+            var context = this_argument || this;
+
+            window.clearTimeout(timer);
+
+            timer = window.setTimeout(() => {
```

Comment:
@punalsc is this resolved? I don't see any changes or your comments on this matter.